modules/mobile-agents/android/jax-rs/src/main/webapp/WEB-INF/cxf-servlet.xml

```diff
@@ -32,7 +32,7 @@
             <ref bean="errorHandler"/>
         </jaxrs:providers>
     </jaxrs:server>
-    <jaxrs:server id="deviceManagementService" address="/devices">
+    <jaxrs:server id="deviceManagementService" address="/device">
```

Comment:
/devices can be a better name than /device

For ex: When getting all devices, A GET call to /devices sounds much better than a GET call to /device